rllib/execution/metric_ops.py

```diff
@@ -118,9 +118,9 @@ class OncePerTimeInterval:
 
     Examples:
         >>> throttled_op = train_op.filter(OncePerTimeInterval(5))
-        >>> start = time.time()
+        >>> start = time.perf_counter()
         >>> next(throttled_op)
-        >>> print(time.time() - start)
+        >>> print(time.perf_counter() - start)
```

Comment:
I'm not sure if the comment on the output of this `print` statement still applies.